frontend-ssr/components/cli_git/git_commands_cc.tsx

```diff
@@ -26,50 +70,79 @@ export default class MatchCommandClassComponent extends React.Component<Props, S
+    } = this.state;
 
     let isMatch = false;
-    if (commands[currentCommandIndex] instanceof RegExp) {
-      isMatch = (commands[currentCommandIndex] as RegExp).test(gitCommand);
+
+    if (commands[currentIndex] instanceof RegExp) {
+      isMatch = (commands[currentIndex] as RegExp).test(gitCommand);
     } else {
-      isMatch = commands[currentCommandIndex] === gitCommand.trim().toLowerCase();
+      isMatch = commands[currentIndex] === gitCommand.trim().toLowerCase();
     }
 
+    const cloneCommandMessages = { ...commandMessages[currentIndex] };
+    const cloneSuccessMessages = { ...successMessages[currentIndex] };
+    const cloneErrorMessages = { ...errorMessages[currentIndexErr] };
+
+    cloneCommandMessages.text = gitCommand;
+    cloneCommandMessages.id = uuidv4();
+    cloneErrorMessages.id = uuidv4();
+    cloneSuccessMessages.id = uuidv4();
+
     if (!isMatch) {
-      this.setState({
-        message: 'please try again',
-        buttonText: 'Next',
-      });
-    } else if (currentCommandIndex === commands.length - 1) {
-      this.setState({
-        message: 'Congratulations!',
-        buttonText: 'Done',
-      });
+      this.setState(({ messages }) => ({
+        messages: [...messages, cloneCommandMessages, cloneErrorMessages],
+      }));
+    } else if (currentIndex === commands.length - 1) {
+      this.setState(({ messages }) => ({
+        messages: [...messages, cloneCommandMessages, cloneSuccessMessages],
+        currentIndex: 0,
+      }));
     } else {
-      this.setState((state) => ({
-        currentCommandIndex: state.currentCommandIndex + 1,
+      this.setState(({ messages }) => ({
+        messages: [...messages, cloneCommandMessages, cloneSuccessMessages],
+        currentIndex: currentIndex + 1,
       }));
     }
     formEl.reset();
   }
 
   render() {
-    const { message, buttonText } = this.state;
+    const { messages } = this.state;
 
     return (
-      <form onSubmit={this.handleSubmit}>
-        <input
-          type="text"
-          name="gitCommand"
-          required
-        />
-        <br />
-        <button type="submit">
-          {buttonText}
-        </button>
-        <br />
-        {message}
-      </form>
+      <>
+        <div className={styles.commands}>
+          <div
+            className={styles.commands__cli}
+          >
+            {messages.map((message) => (
+              <p key={message.id} className={styles[message.type]}>
+                {message.text}
+                {' '}
+                {message.time}
+              </p>
+            ))}
+          </div>
+          <form
+            className={styles.form}
+            onSubmit={this.handleSubmit}
+          >
+            <input
+              className={styles.form__input}
+              type="text"
+              name="gitCommand"
+              required
+            />
+            <span />
+            <button className={styles.btn} type="submit">
```

Comment:
This button is hidden. It should be visible, in case someone sends from a tablet/mobile. Also, rembmer the button should change it's text, as it does now.